lib/chef/resource/support/client.erb

```diff
@@ -0,0 +1,64 @@
+<% %w(node_name
+      chef_license
+      chef_server_url
+      event_loggers
+      file_backup_path
+      file_cache_path
+      file_staging_uses_destdir
+      formatters
+      http_proxy
+      https_proxy
+      ftp_proxy
+      log_level
+      log_location
+      minimal_ohai
+      named_run_list
+      no_proxy
+      ohai_disabled_plugins
+      ohai_optional_plugins
+      pid_file
+      policy_group
+      policy_name
+      run_path
+      ssl_verify_mode).each do |prop| -%>
```

Comment:
You could probably `.map { |prop| instance_variable_get('@' + prop) }` to avoid calling `eval` multiple times.

Reply:
yeah definitely convert the eval stuff to instance_variable_get

Reply:
TIL ﻿instance_variable_get. I updated it for that.